### PR TITLE
Add test to approximately format

### PR DESCRIPTION
**Pull request by romulocintra**
Summary:
Add test for FormatApproximately source in formatRangeToParts

Closes https://github.com/tc39/test262/issues/3764

### CODE REVIEW

File: test/intl402/NumberFormat/prototype/formatRangeToParts/en-US.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ function compare(actual, expected) {
     assert.sameValue(actualEntry.type, expectedEntry.type, `type for entry ${i}`);
     assert.sameValue(actualEntry.value, expectedEntry.value, `value for entry ${i}`);
     assert.sameValue(actualEntry.source, expectedEntry.source, `source for entry ${i}`);
```
[the remark this answers is not in the document]
Comment (Ms2ger) [reply test/intl402/NumberFormat/prototype/formatRangeToParts/en-US.js:25]: Don't we dislike deepEqual?

[the remark this answers is not in the document]
Comment (ptomato) [reply test/intl402/NumberFormat/prototype/formatRangeToParts/en-US.js:25]: https://github.com/tc39/test262/issues/3476

[the remark this answers is not in the document]
Comment (romulocintra) [reply test/intl402/NumberFormat/prototype/formatRangeToParts/en-US.js:25]: Should I apply deepEqual or there is any further changes I should do ?

[the remark this answers is not in the document]
Comment (gibson042) [reply test/intl402/NumberFormat/prototype/formatRangeToParts/en-US.js:25]: The PR looks fine to me with or without adopting that suggestion.

[the remark this answers is not in the document]
Comment (ptomato) [reply test/intl402/NumberFormat/prototype/formatRangeToParts/en-US.js:25]: OK, thanks, let's go with this as is then.

**Comment by romulocintra** [comment]
cc @sffc

**Review by sffc — APPROVED** [review]

**Comment by ptomato** [comment]
I can't find in the notes where the normative PR was presented, but since it's included in the stage 4 PR for NumberFormat V3 I'm assuming it's approved.

**Pull request merged**